jep/1/README.adoc

```diff
@@ -0,0 +1,542 @@
+appropriate alternative list where discussion, review and pronouncement on the
+JEP will occur.
+
+For a JEP to be accepted it must meet certain minimum criteria:
+
+* It must be a clear and complete description of the proposed enhancement.
+* The enhancement must represent a net improvement.
+* The proposed implementation, if applicable, must be solid and must not complicate Jenkins unduly.
+
+Once a JEP has been accepted, the implementation must be completed. The Jenkins
+project values contribution over "talk"
+footnote:[https://jenkins.io/project/governance/#meritocracy], and as such the
+implementation is of utmost importance to moving any proposal (Standards or
+Process) forward. When the implementation is complete and incorporated into the
+appropriate "main" code repository, the status will be changed to "Final".
+
+A JEP can also be assigned status "Deferred". The JEP author or an
+editor can assign the JEP this status when no progress is being made
+on the JEP. Once a JEP is deferred, a JEP editor can re-assign it
+to draft status.
+
+A JEP can also be "Rejected". Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact. The "Withdrawn" status is similar - it means that the JEP author
+themselves has decided that the JEP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a JEP is Accepted, Rejected or Withdrawn, the JEP should be updated
+accordingly. In addition to updating the status field, at the very least
+the Resolution header should be added with a link to the relevant post
+in the jenkinsci-dev@ mailing list archives.
+
+JEPs can also be superseded by a different JEP, rendering the original
+obsolete. This is intended for Informational JEPs, where version 2 of
+an API can replace version 1.
+
+The possible paths of the status of JEPs are as follows:
+
+image::workflow.png[JEP Workflow]
+
+Some Informational and Process JEPs may also have a status of "Active" if they
+are never meant to be completed. E.g. JEP 1 (this JEP).
+
+
+=== JEP Maintenance
+
+In general, Standards track JEPs are no longer modified after they have
+reached the Final state. Once a JEP has been completed, Jenkins developer
+documentation must become the formal documentation of the expected behavior.
+
+Informational and Process JEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the JEP being updated.
+
+
+== What belongs in a successful JEP?
+
+Each JEP should have the following parts:
+
+. **Metadata** - table containing metadata about the JEP, including the JEP
+  number, a short descriptive title, the names, and optionally the contact info
+  for each author, etc.
+. **Abstract** - short (200 word) description of the technical issue
+  being addressed.
+. **Specification** - The technical specification should describe the
```

Comment:
Specification should come after motivation.

Reply:
This comment requires more explanation if we're to make this change. This is the ordering copied from PEP-1, and I personally find Abstract/Specification to be a reasonable first couple sections to review of a technical design document.

Reply:
@rtyler Thinking about it a bit more, I agree.  Putting the Motivation first helps set context for those unfamiliar with it.  But Motivation is fluff for those already aware of the it.